include/nlohmann/detail/meta/type_traits.hpp

```diff
@@ -8,10 +8,11 @@
 
```

Comment:
@colbychaskell was this file auto-generated or formatted by the linter in CI?

if not, there seem to be a lot of formatting changes only (e.g. `{}` on new line) which add a lot of noise for reviewing. maybe avoid the formatting only changes? wdyt?

Reply:
Thanks, I didn't see this. I must've accidentally run clang-format in vscode before committing. 

Fixed this up now!